cpu/ppc64_cpu_test.py

```diff
@@ -91,11 +91,16 @@ def test(self):
             self.threads_per_core()
             self.smt_snoozedelay()
             self.dscr()
+
+        self.smt_loop()
+
         if self.failures > 0:
             self.log.debug("Number of failures is %s" % self.failures)
             self.log.debug(self.failure_message)
             self.fail()
 
+        process.system_output("dmesg")
```

Comment:
I wanted to collect the dmesg log so that if that I can capture any errors due to this test. I infact had added a dmesg -c in the setup but then decided against it as it might affect logs when this test case is run with other avocado tests in a batch.